src/helpers/discord_utils.ts

```diff
@@ -217,75 +211,115 @@ export async function sendEndRoundMessage(messageContext: MessageContext,
  * @param message - The Message object
  * @param guildPreference - The corresponding GuildPreference
  * @param updatedOption - Specifies which GameOption was modified
- * @param footerText - The footer text
+ * @param footerText - The footer text (shows prefix when bot is pinged)
```

Comment:
this footer text can be anything, doesnt make to document it for a specific case